source/mainnet/smart-contracts/tutorials/piggy-bank/preparing.rst

```diff
@@ -0,0 +1,231 @@
+
+.. image:: ./images/pb_tutorial_14.png
+   :width: 100 %
+
+Let's check if everything is connected correctly by displaying the best/latest block.
+
+.. code-block:: console
+
+   $./concordium-client block show --grpc-ip <IP Address of Your Instance> --grpc-port 10001
+
+You should see some block data output.
+
+.. image:: ./images/pb_tutorial_17.png
+   :width: 100 %
+
+
+Create a mobile wallet account on testnet
+=========================================
+
+You can create an account with any of the officially provided wallets from Concordium. This tutorial focuses on the mobile wallet.
+
+You can download the mobile wallet package from the :ref:`installation page<downloads-mobile-wallet-testnet>`.
+If you follow the instruction in the mobile app you are asked to name your account and to name your initial identity card. This tutorial uses `Concordium` for the account name and `Account1` for the identity card name but feel free to chose your own naming.
+
+
+.. image:: ./images/pb_tutorial_1.png
+   :width: 20 %
+
+.. image:: ./images/pb_tutorial_2.png
+   :width: 20 %
+
+.. image:: ./images/pb_tutorial_3.png
+   :width: 20 %
+
+
+You don't have to provide an ID to create an account on testnet when selecting `Concordium testnet IP`. This gives you an example for a dummy identity that you can customise or use as it is. Dummy identities are meant for testnet testing only.
+
+.. image:: ./images/pb_tutorial_4.png
+   :width: 20 %
+
+.. Note::
+   On mainnet you will have to provide your personal ID to an identity provider to get a mainnet account.
+
+You also have to request some testnet CCD. The mobile app wallet has a button that you can use to request 2000 testnet CCD.
+
+.. image:: ./images/pb_tutorial_5.png
+   :width: 20.5 %
+.. image:: ./images/pb_tutorial_6.png
+   :width: 20 %
+
+.. Note::
+   Some CCD on your testnet account is needed later when sending transactions from your account to the testnet blockchain.
+
+Create a backup of your wallet by clicking the `Backup` button in the mobile app. Save the file `concordium-backup.concordiumwallet` for now because it will be used in the next section.
+
+.. image:: ./images/pb_tutorial_7.png
+   :width: 20 %
+
+.. Note::
+   Please remember the `export password` that you used for creating the backup file. The password is needed later when importing your account key into the testnet node.
+
+
+Import your mobile wallet account key to your testnet node
+==========================================================
+
+You are ready now to import your keys to your testnet node. Let's transfer your wallet backup file (meaning the file `concordium-backup.concordiumwallet`) to the place where you are running your concordium client tool.
```

Comment:
Keys are not **imported into the node**. They are imported into the concordium-client configuration which is entirely distinct from the node.